scene/gui/gradient_edit.cpp

```diff
@@ -54,6 +54,9 @@ GradientEdit::GradientEdit() {
 
 	checker = Ref<ImageTexture>(memnew(ImageTexture));
 	Ref<Image> img = memnew(Image(checker_bg_png));
+	checker->create_from_image(img);
+	set_texture_repeat(TEXTURE_REPEAT_ENABLED); // Repeat background checkerboard
+	set_texture_filter(TEXTURE_FILTER_NEAREST); // Do not blur background checkerboard with filter
```

Comment:
I wanted to use a `tile` parameter, but that doesn't exist on `draw_colored_polygon`